mlflow/openai/_openai_autolog.py

```diff
@@ -101,7 +114,15 @@ def patched_call(original, self, *args, **kwargs):
             run_id = run.info.run_id
 
     with disable_autologging():
-        result = original(self, *args, **kwargs)
+        with mlflow.start_span(
+            name=self.__class__.__name__,
+            span_type=_get_span_type(self.__class__),
+        ) as span:
+            # openai does not accept positional arguments
+            # so we do not need to worry about it for now
+            span.set_inputs(kwargs)
+            result = original(self, *args, **kwargs)
+            span.set_outputs(result)
```

Comment:
Can we add `log_traces` to `mlflow.openai.auotlog` as a flag to disable/enable trace logging?

Reply:
sure, added!